src/composeappmanager.cc

```diff
@@ -67,7 +67,7 @@ ComposeAppManager::ComposeAppManager(const PackageConfig& pconfig, const Bootloa
     LOG_DEBUG << "Failed to initialize Compose App Tree (ostree) at " << cfg_.apps_tree << ". Error: " << exc.what();
   }
 
-  const auto& current_apps = getApps(getCurrent());
+  const auto& current_apps = getApps(OstreeManager::getCurrent());
```

Comment:
Is the idea here that once `ComposeAppManager::getCurrent` handles the "TODO", all we really need from this method is the logic of its parent class?